zaza/openstack/charm_tests/nova/tests.py

```diff
@@ -384,6 +385,56 @@ def test_902_quota_settings(self):
             alternate_entry,
             self.services)
 
+    def test_903_enable_quota_count_usage_from_placement(self):
+        """Verify that quota-count-usage-from-placement is propagated.
+
+        Change quota-count-usage-from-placement and assert that nova
+        configuration file is updated and the services are restarted.
+        This parameter is not supported for releases<Train. In those
+        cases assert that nova configuration file is not updated.
+        """
+        # Expected default and alternate values
+        current_value = zaza.model.get_application_config(
+            'nova-cloud-controller')['quota-count-usage-from-placement']
+        try:
+            current_value = current_value['value']
+        except KeyError:
+            current_value = False
+        new_value = True
+        new_value_str = str(new_value).title()
```

Comment:
What will the test do if the `current_value` is `True`?

Reply:
Wrong assumption that cloud will be deployed with default values. Corrected the functional test case to perform tests irrespective of juju config of quota-count-usage-from-placement on the deployed cloud. Please re-review.